docs/guides/dev_guide.md

```diff
@@ -188,20 +198,20 @@ More information: [Protocol-rfc#0018 - Transaction Payload](https://github.com/l
 `SignedTransaction` is a payload type that is used to transfer value-based messages as `UTXO`. It changes the ledger state as old `outputs` are being spent (replaced) and new `outputs` are being created.
 
 Each `SignedTransaction` includes the following set of information:
-* `inputs` - a list of valid `outputs` that should be used to fund the given transaction. Outputs are uniquely referenced via previous `transaction_id` and inner `index`. At least one output has to be given with enough balance to source all `outputs` of the given message
-* `outputs` - a list of IOTA address(es) and related amount(s) the input `outputs` should be split among. Based on this information, new `UTXO` outputs are being created
-* `unlock_blocks` - it includes a transaction signature(s) (currently based on `Ed25519` scheme) that proves token ownership based on a valid seed
+* `inputs` - a list of valid `outputs` that should be used to fund the given transaction. Outputs are uniquely referenced via previous `transaction_id` and inner `index`. At least one output has to be given with enough balance to source all `outputs` of the given message.
+* `outputs` - a list of IOTA address(es) and related amount(s) the input `outputs` should be split among. Based on this information, new `UTXO` outputs are being created.
+* `unlock_blocks` - it includes a transaction signature(s) (currently based on `Ed25519` scheme) that proves token ownership based on a valid seed.
```

Comment:
* `unlock_blocks`:  it includes a transaction signature(s) (currently based on `Ed25519` scheme) that prove(s) token ownership based on a valid seed.